python/cudf/cudf/tests/test_binops.py

```diff
@@ -2103,10 +2139,16 @@ def test_binops_with_NA_consistent(dtype, op):
         ),
     ],
 )
-def test_binops_decimal(args):
-    op, lhs, l_dtype, rhs, r_dtype, expect, expect_dtype = args
-
-    a = utils._decimal_series(lhs, l_dtype)
+def test_binops_decimal(op, lhs, l_dtype, rhs, r_dtype, expect, expect_dtype):
+
+    if l_dtype is None:
```

Comment:
Can we change this to just interpret scalar values as `Scalar`?
```suggestion
    if isinstance(lhs, (int, float)):
```
Then define `l_dtype` as `cudf.Decimal64Dtype(scale=2, precision=6)` or similar for the scalar values, instead of using `None`.

Then the comment below is unnecessary and the result is `cudf.Scalar(2, cudf.Decimal64Dtype(scale=2, precision=6))` or similar.

Reply:
Done 👍